Source/WPEFramework/json/Controller.json

```diff
@@ -310,6 +310,95 @@
         }
       ]
     },
+    "Controller.1.resume": {
```

Comment:
Think we should also regenerate the documentation from this (but that is not super urgent indeed)